internal/controller/datadogagent/feature/admissioncontroller/feature.go

```diff
@@ -69,6 +71,10 @@ type AgentSidecarInjectionConfig struct {
 	profiles                         []*v2alpha1.Profile
 }
 
+type KubernetesAdmissionEventConfig struct {
+	enabled bool
```

Comment:
Created struct for k8s admin event config bc there is potential future use case for users to customize the feature such as having more control in which events are recorded